lib/internal/worker.js

```diff
@@ -505,12 +507,80 @@ function pipeWithoutWarning(source, dest) {
   dest._maxListeners = destMaxListeners;
 }
 
+class LockManager {
+  request(name, options, callback) {
+    if (callback === undefined) {
```

Comment:
I would prefer this to be consistent with Node.js terminology. A callback for Node.js is an error-back. This would be confusing. This would also apply to our docs.

Reply:
I agree. While I generally prefer to keep naming the same or as same as possible to specs, in this case not calling this `callback` makes sense. Also, on the off chance that someone decided to try to wrap this in `util.promisify()` because it *looks* like a callback, it may make sense to have a custom promisify implementation for this.

Reply:
util.promisify makes no sense here, it already returns a promise. i will rename the variable.

Reply:
Yes, I know that `util.promisify()` makes no sense here, but that doesn't mean users won't *try*. Being defensive by providing a custom promisify implementation that skips wrapping and returns the actual promise makes sense... or, throw within the custom promisify to indicate that it doesn't make sense.